db/seeds.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
```

Comment:
- [ ] Kindly create 5 greetings (messages), this should be done using the seed file.

![messages](https://github.com/Dadadon/hello-rails-back-end/assets/53124548/a7d64d9c-4fba-4e91-96aa-363aa577b87a)